modules/lib/lib-node/src/main/resources/lib/xp/examples/node/query-dsl-sort.js

```diff
@@ -0,0 +1,56 @@
+var nodeLib = require('/lib/xp/node');
+var assert = require('/lib/xp/testing');
+
+/* global log */
+
+var repo = nodeLib.connect({
+    repoId: 'com.enonic.cms.default',
+    branch: 'master'
+});
+
+// BEGIN
+// Query content using aggregations.
+var result = repo.query({
+    start: 0,
+    count: 2,
+    query: {
+        boolean: {
+            must: {
+                range: {
+                    field: 'startTime',
+                    type: 'dateTime',
+                    from: '2016-10-11T14:38:54.454Z'
+                }
+            }
+        }
+    },
+    sort: {field: '_name', direction: 'ASC'}
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8706041)
```suggestion
    sort: {field: "_name", direction: 'ASC'}
```